<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCoolBar</class>
 <widget class="QWidget" name="CCoolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>51</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CCoolBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="m_docsFilter" native="true">
     <property name="minimumSize">
      <size>
       <width>158</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>148</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="m_filter">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_leftPlaceHolder" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="m_docsToolBar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="m_btnNewFile">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>New File</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/document-new.png</normaloff>:/res/icons/document-new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnOpenFile">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Open File</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/document-open.png</normaloff>:/res/icons/document-open.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnSaveFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Save File</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/document-save.png</normaloff>:/res/icons/document-save.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnExport">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/document-export.png</normaloff>:/res/icons/document-export.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnUndo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Undo</string>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/edit-undo.png</normaloff>:/res/icons/edit-undo.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnRedo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Redo</string>
        </property>
        <property name="text">
         <string>Redo</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/edit-redo.png</normaloff>:/res/icons/edit-redo.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tools" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_btnFreeTopic">
           <property name="toolTip">
            <string>Insert Free Topic</string>
           </property>
           <property name="text">
            <string>Topic</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/global-free-topic.png</normaloff>:/res/icons/global-free-topic.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnCursor">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Cursor</string>
           </property>
           <property name="text">
            <string>Cursor</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/global-cursor.png</normaloff>:/res/icons/global-cursor.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnText">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add Text</string>
           </property>
           <property name="text">
            <string>Text</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/edit-text.png</normaloff>:/res/icons/edit-text.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnConnection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add New Edge</string>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/node-edge.png</normaloff>:/res/icons/node-edge.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnFullScreen">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start Presentation</string>
        </property>
        <property name="text">
         <string>Full Screen</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-start.png</normaloff>:/res/icons/presentation-start.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnBrainStorm">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start Brain Storm</string>
        </property>
        <property name="text">
         <string>Brain Storm</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-edit.png</normaloff>:/res/icons/presentation-edit.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_rightPlaceHolder" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="m_docsCtrl" native="true">
     <property name="minimumSize">
      <size>
       <width>158</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>148</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="m_rightPartPlaceHolder" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_2" native="true"/>
        </item>
        <item>
         <widget class="QToolButton" name="m_btnPrev">
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Previous File</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/mac/res.qrc">
            <normaloff>:/res/icons/document-page-previous.png</normaloff>:/res/icons/document-page-previous.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_btnNext">
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Next File</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/mac/res.qrc">
            <normaloff>:/res/icons/document-page-next.png</normaloff>:/res/icons/document-page-next.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_btnOpenedFiles">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>All Opened Files</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../res/mac/res.qrc">
            <normaloff>:/res/icons/document-list.png</normaloff>:/res/icons/document-list.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_btnClose">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Close File</string>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/mac/res.qrc">
            <normaloff>:/res/icons/document-close.png</normaloff>:/res/icons/document-close.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
